docs/custom_domains.rst

```diff
@@ -1,11 +1,9 @@
 Custom Domains
 ==============
 
-.. note:: These setup directions are for our community site.
-          If you want to setup a custom domain under our `commercial hosting`_,
-          please read our :ref:`commercial documentation <commercial/custom_domains:Custom Domains>`.
-
-.. _commercial hosting: https://readthedocs.com
+.. note:: These directions are for projects hosted on our community site.
+          If you want to setup a custom domain on `Read the Docs for Business <https://readthedocs.com/>`_,
```

Comment:
Sometimes we link "Read the Docs for Business" to readthedocs.com and some others to `<commercial/index>`. Is that on purpose?

Reply:
In general, I tried to link directly to readthedocs.com from subpages of `commercial` and linked to `commercial/index` on other pages. This page was the only special one where I linked to both (actually it already did that).